pages/apply/not-eligible.tsx

```diff
@@ -17,7 +17,7 @@ export default function NotEligible(): JSX.Element {
     (store) => store.application.mainApplicant
   );
   const [isEligible, reasons] = useMemo(
-    () => (mainApplicant && checkEligible(mainApplicant)) ?? [],
+    () => (mainApplicant && checkEligible(mainApplicant, true)) ?? [],
```

Comment:
The other pages are checking each member of the application, does this need updating to match?